doc/latex.rst

```diff
@@ -940,32 +940,31 @@ The color used in the above example is available from having passed the
 
     Default: ``\fboxrule``
 
-``shadowsep``
-    The separation between contents and frame for contents_ and
-    :dudir:`topic` boxes.
-
-    See :ref:`additionalcss` for the alias ``div.topic_padding``.
+.. important::
 
-    Default: ``5pt``
+   At 8.1.0 it became possible to style separately the :dudir:`topic`,
+   contents_, and :dudir:`sidebar` directives, and each acquired new defaults.
+   See :ref:`additionalcss`.
+   The ``shadowsep``, ``shadowsize`` and ``shadowrule`` keys are kept for
+   backward compatibility only.
 
-``shadowsize``
-    The width of the lateral "shadow" to the right and bottom.
+``shadowsep``
+    This legacy option sets the padding (same in all directions) simultaneously
+    for the :dudir:`topic`, contents_, and :dudir:`sidebar` directives.
 
-    See :ref:`additionalcss` for ``div.topic_box-shadow`` which allows to
-    configure separately the widths of the vertical and horizontal shadows.
+    .. versionchanged:: 8.1.0 New separate defaults. See :ref:`additionalcss`.
```

Comment:
> I don't know whether it should be a versionchanged or a deprecated since they are kept for backward compatibility.
> 

I will check that.
> Also, I would separate 8.1.0 from "New":
> 
> ```
> .. versionchanged:: 8.1.0
>    New...
> ```
> 

Formerly I think I used

```
.. versionchanged:: X.y.z

   description
```
and one can see many examples in doc/latex.rst.  Now I am very confused about how this directive is supposed to be used.

> Same obversation for the other `.. version*` directives that you added.

Reply:
All those ``versionchanged`` removed in next push, and a clearer explanation in "important" box before.